### PR TITLE
[Bulk] Validation-Missing-Alttext-5

### PR DESCRIPTION
**Pull request by Roudabeh1348**
This PR fixes 82 missing alt text issues in iis-docs file.

### CODE REVIEW

File: iis/troubleshoot/using-failed-request-tracing/troubleshooting-failed-requests-using-tracing-in-iis-85.md [hunk 1]
```diff
@@ -154,21 +154,21 @@ In this task, you will generate a failed request and view the resulting trace lo
...
+    [![Screenshot of Internet Explorer dialog box with Continue to prompt when website content is blocked option selected.](troubleshooting-failed-requests-using-tracing-in-iis-85/_static/image27.jpg)](troubleshooting-failed-requests-using-tracing-in-iis-85/_static/image26.jpg)
 3. In the **Internet Explorer** dialog box, add **about:internet** to the list of trusted sites by selecting the **Tools** menu, selecting **Internet Options**, clicking the **Security** tab, selecting **Trusted Zone**, and then selecting **Sites**. This allows the XSL to work. You will see the following after adding **about:internet** to the list of trusted sites:  
 
-    [![](troubleshooting-failed-requests-using-tracing-in-iis-85/_static/image29.jpg)](troubleshooting-failed-requests-using-tracing-in-iis-85/_static/image28.jpg)
+    [![Screenshot of Request Summary page with the Errors & Warnings table displaying columns for Severity, Event and module Name.](troubleshooting-failed-requests-using-tracing-in-iis-85/_static/image29.jpg)](troubleshooting-failed-requests-using-tracing-in-iis-85/_static/image28.jpg)
```
Comment (RedZephyr13) [inline iis/troubleshoot/using-failed-request-tracing/troubleshooting-failed-requests-using-tracing-in-iis-85.md:171]: Let's replace "&" with "and."

File: iis/configuration/system.webServer/management/authentication/credentials/add.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ The default installation of IIS 7 and later does not include the **Management Se
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows Features on or off**.
 3. Expand **Internet Information Services**, then **Web Management Tool**.
 4. Select **IIS Management Service**, and then click **OK**.   
-    [![](add/_static/image8.png)](add/_static/image7.png)
+    [![Screenshot displays management Service dialog box with Enable remote connections box selected.](add/_static/image8.png)](add/_static/image7.png)
```
[the remark this answers is not in the document]
Comment (Roudabeh1348) [reply iis/configuration/system.webServer/management/authentication/credentials/add.md:73]: Done!

File: iis/troubleshoot/using-failed-request-tracing/troubleshooting-failed-requests-using-tracing-in-iis-85.md (hunk 1 above)
Comment (Roudabeh1348) [reply iis/troubleshoot/using-failed-request-tracing/troubleshooting-failed-requests-using-tracing-in-iis-85.md:171]: fixed.

**Comment by Roudabeh1348** [comment]
@Rick-Anderson @mcleblanc This PR fixes validation warnings/suggestions per a platform-wide push to increase content health scores. Please review at your convenience and sign off or merge if you approve or let me know if I need to make any other changes.  cc: @MonicaRush @snoviking

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**